lib/slipstream/connection/impl.ex

```diff
@@ -17,18 +17,22 @@ defmodule Slipstream.Connection.Impl do
 
   def http_connect(config) do
     Mint.HTTP.connect(
-      map_scheme(config.uri.scheme),
+      map_http_scheme(config.uri.scheme),
       config.uri.host,
       config.uri.port,
       config.mint_opts
     )
   end
 
-  defp map_scheme("wss"), do: :https
-  defp map_scheme(_), do: :http
+  defp map_http_scheme("wss"), do: :https
+  defp map_http_scheme(_), do: :http
```

Comment:
Stupid q, why not https everywhere?

Reply:
I would probably prefer that within our own stuff but I think it's good to have it be configurable (for example if someone is running something locally and doesn't want to set up a self-signed cert)